.github/workflows/build_images.yml

```diff
@@ -2,38 +2,65 @@ name: Build Images
 
 concurrency: ${{ github.ref }}
 
-on:
-  workflow_dispatch:
+on: 
   push:
     branches: [main]
     paths:
       - 'images/**'
       - 'scripts/**'
-      # - 'builder/**'
+  workflow_dispatch:
 
 jobs:
+
   build:
-    # runs-on: windows-latest
     runs-on: ubuntu-latest
 
-    if: "!contains(join(github.event.commits.*.message), '+nobuild')"
-
```

Comment:
This also seems a good condition. So we add `nobuild` to a commit message to stop running this github action.